src/redux/configureStore.js

```diff
@@ -1,29 +1,27 @@
+import { applyMiddleware, compose, createStore } from 'redux'
+import { syncHistory } from 'redux-simple-router'
 import thunk from 'redux-thunk'
 import rootReducer from './rootReducer'
-import {
-  applyMiddleware,
-  compose,
-  createStore
-} from 'redux'
 
-export default function configureStore (initialState) {
-  let createStoreWithMiddleware
-  const middleware = applyMiddleware(thunk)
+function withDevTools (middleware) {
+  const devTools = window.devToolsExtension
+    ? window.devToolsExtension()
+    : require('containers/DevTools').default.instrument()
```

Comment:
@maxekman no, it will not—require is referencing the default key on imported modules... did you actually merge this PR and encounter an error?